### PR TITLE
Bug + Custom Update.

### PR DESCRIPTION
**Pull request by CozarXXVIII**
Curse HoverItem fixed. 
Removed + from Dialog stat page.

Custom:
Item Description label.
Drop item rarity colours added.
{$GUILDNAME} added.

### CODE REVIEW

File: Client/MirScenes/GameScene.cs [hunk 1]
```diff
@@ -5926,7 +5926,7 @@ public MirControl AttackInfoLabel(UserItem item, bool Inspect = false)
             maxValue = 0;
             addValue = (!HoverItem.Info.NeedIdentify || HoverItem.Identified) ? HoverItem.Luck : 0;
 
-            if (minValue > 0 || addValue != 0)
+            if (minValue != 0 || addValue > 0)
```
Comment (Suprcode) [inline Client/MirScenes/GameScene.cs]: add value should still be !=

File: Client/MirScenes/GameScene.cs [hunk 2]
```diff
@@ -7908,10 +7908,24 @@ public MirControl StoryInfoLabel(UserItem item, bool Inspect = false)
...
+                };
+
+                ItemLabel.Size = new Size(Math.Max(ItemLabel.Size.Width, IDLabel.DisplayRectangle.Right + 4),
+                    Math.Max(ItemLabel.Size.Height, IDLabel.DisplayRectangle.Bottom));
+
                 MirLabel TOOLTIPLabel = new MirLabel
                 {
                     AutoSize = true,
-                    ForeColour = Color.Goldenrod,
+                    ForeColour = Color.Khaki,
```
Comment (Suprcode) [inline Client/MirScenes/GameScene.cs]: why the change in tip colour? do you have a screenshot of what this looks like?

File: Client/MirScenes/GameScene.cs [hunk 3]
```diff
@@ -7908,10 +7908,24 @@ public MirControl StoryInfoLabel(UserItem item, bool Inspect = false)
             if (!string.IsNullOrEmpty(HoverItem.Info.ToolTip))
             {
                 count++;
+
+                MirLabel IDLabel = new MirLabel
```
Comment (Suprcode) [inline Client/MirScenes/GameScene.cs]: what is this IDLabel?

**Comment by daneo1989** [comment]
Screenshot should answer all the quests above, 
untitled1
